packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
+   selectedCategories TEXT,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this the selected category names or ids? if names - what happens if the category gets renamed? Will it break all the reports?

Also I think `hiddenCategories` would be better naming here.

Reply:
They are "type CategoryEntity" elements. The lookup is done by id so renaming won't break anything.

The selected Categories in this list are the ones that are shown in the graph so I think "hiddenCategories" would be mislabeled. It's also called "selectedCategories" everywhere else in the code so having a differnt name here would be confusing to follow for any future maintenance.